setup.cfg

```diff
@@ -16,7 +16,11 @@ exclude = .tox
 [tool:pytest]
 minversion = 5
 testpaths = tests
-addopts = --cov
+addopts =
+    --cov
+    -m "not workflow"
```

Comment:
@LinguList  Here we exclude workflow tests from any "regular" `pytest` run.

Reply:
very useful to know, thanks!